public/Invoke-DbaDbLogShipping.ps1

```diff
@@ -1376,12 +1376,6 @@ function Invoke-DbaDbLogShipping {
                     }
                 }
 
-                # Check the primary monitor server
-                if ($Force -and (-not $PrimaryMonitorServer -or [string]$PrimaryMonitorServer -eq '' -or $null -eq $PrimaryMonitorServer)) {
-                    Write-Message -Message "Setting monitor server for primary server to $SourceSqlInstance." -Level Verbose
-                    $PrimaryMonitorServer = $SourceSqlInstance
-                }
-
```

Comment:
This is breaking change is it not?

Reply:
Looks like it but this is a good time, as we're about to release a new version.